### PR TITLE
aria-rowcount expectations, test results

### PR DESCRIPTION
**Pull request by RavenAlly**

### CODE REVIEW

File: data/tech/aria/aria-rowcount_attribute.json [hunk 1]
```diff
@@ -9,6 +9,46 @@
...
+        "vc": "NA"
+      },
+      "css_target": "*[aria-rowcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "do_not_calculate_unknown_row_count",
+      "title": "convey a row count when total row count is unknown",
```
Comment (mfairchild365) [inline data/tech/aria/aria-rowcount_attribute.json:41]: Awesome expectation, good work! I had forgotten that ARIA mentions this.

My only concern is that the expectation doesn't mention anything about "-1" being the magical value to indicate this. We could add a "notes" property to the expectation and explain this. Thoughts?

File: data/tech/aria/aria-rowcount_attribute.json [hunk 2]
```diff
@@ -9,6 +9,46 @@
...
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "*[aria-rowcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "convey_row_count_hidden_rows",
```
Comment (mfairchild365) [inline data/tech/aria/aria-rowcount_attribute.json:27]: How do you feel about reducing `convey_row_count_all_rows` and `convey_row_count_hidden_rows` to one expectation? The results are the same, and the ARIA spec states "If all of the rows are present in the DOM, it is not necessary to set this attribute as the user agent can automatically calculate the total number of rows." That makes me think that `convey_row_count_all_rows` might not be necessary for us to test.

**Review by mfairchild365 — COMMENTED** [review]
Fantastic work! thank you so much! Just a few questions/comments.

File: data/tests/html/aria/aria-rowcount.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
```
[the remark this answers is not in the document]
Comment (RavenAlly) [reply data/tests/html/aria/aria-rowcount.html]: Well that's embarrassing... 🤦‍♀️

File: data/tech/aria/aria-rowcount_attribute.json (hunk 1 above)
Comment (RavenAlly) [reply data/tech/aria/aria-rowcount_attribute.json:28]: Added note verbatim from the spec. ✔️

File: data/tech/aria/aria-rowcount_attribute.json (hunk 2 above)
Comment (RavenAlly) [reply data/tech/aria/aria-rowcount_attribute.json:14]: Removed the all_rows expectation and test results. ✔️

File: data/tests/html/aria/aria-rowcount.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (mfairchild365) [reply data/tests/html/aria/aria-rowcount.html]: Naw, I make mistakes like this all the time. We are all human, and that's why PR reviews are helpful.

**Comment by mfairchild365** [comment]
Thank you so much!

**Pull request merged**